evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -874,9 +881,9 @@ def test_generate_pipeline_example(
     problem_configuration = (
         {
             "time_index": "date",
-            "gap": 1,
-            "max_delay": 1,
-            "forecast_horizon": 3,
+            "gap": 0,
+            "max_delay": 2,
+            "forecast_horizon": 1,
```

Comment:
Might be a dumb question but any reason why we changed the problem configuration to this particular set of numbers?